File: 3387-minimum-operations-to-make-median-of-array-equal-to-k/minimum-operations-to-make-median-of-array-equal-to-k.java
class Solution {
    public long minOperationsToMakeMedianK(int[] nums, int k) {
        Arrays.sort(nums);
        int mid = (int)nums.length/2;
        long cost = 0;
        if(nums[mid] == k){
            return 0;
        }
        else if(nums[mid] <k){
            while(mid<nums.length && nums[mid]<k){
                cost += k-nums[mid++];
            }
        }
        else{
            while(mid>-1 && nums[mid]>k){
                cost += nums[mid--]-k;
            }
        }
        return cost;
        
    }
}

File: 3387-minimum-operations-to-make-median-of-array-equal-to-k/minimum-operations-to-make-median-of-array-equal-to-k.py
class Solution:
    def minOperationsToMakeMedianK(self, nums: List[int], k: int) -> int:
        nums.sort()
        mid = len(nums)//2
        if(nums[mid] == k):
            return 0
        elif(nums[mid]<k):
            cost = 0
            while(mid<len(nums) and nums[mid]<k):
                cost += abs(nums[mid]-k)
                mid += 1
        else:
            cost = 0
            while(mid>-1 and nums[mid]>k):
                cost += abs(nums[mid]-k)
                mid -= 1
        return cost
            
        